modules/govuk_jenkins/templates/jobs/bouncer_cdn_redirect.yaml.erb

```diff
@@ -0,0 +1,64 @@
+---
+- scm:
+    name: cdn-configs_Bouncer_CDN
+    scm:
+        - git:
+            url: git@github.com:alphagov/cdn-configs.git
+            branches:
+              - master
+
+- job:
+    name: Bouncer_CDN
+    display-name: Bouncer_CDN
+    project-type: freestyle
+    description: |
+        <a href='https://deploy.blue.production.govuk.digital/job/Bouncer_CDN/'> 
+            <img src='https://www.futurefinance.com/img/blogimages/2015/12/We_have_moved1.png'
+        </a>
+        <p style=''>
+            <strong>So we have moved to a new location click on the image or link to get redirected to the correct Jenkins instance.
+        </p>
+    properties:
+        - build-discarder:
+            days-to-keep: 30
+            artifact-num-to-keep: 5
+        - github:
+            url: https://github.com/alphagov/cdn-configs/
```

Comment:
We do not need any github properties in this - the job section should be the only one populated (so this is not accidentally able to do something :)